src/openfl/_internal/renderer/canvas/CanvasTextField.hx

```diff
@@ -68,6 +68,10 @@ class CanvasTextField {
 			var width = graphics.__width;
 			var height = graphics.__height;
 			
+			if (graphics.__bitmap != null) {
+				graphics.__bitmap.disposeImage ();
+			}
+			
```

Comment:
This is similar to the `CanvasGraphics` case, I believe.

Reply:
Yes, same comment as above ^^